pkg/pool/operations/status.go

```diff
@@ -44,9 +45,9 @@ func GetPropertyValue(poolName, property string) (string, error) {
 	return outStr[0], nil
 }
 
-// GetListOfPropertyValues will return value list for given property list
+// GetPoolListPropertyValues will return value list for given property list
 // NOTE: It will return the property values in the same order as property list
-func GetListOfPropertyValues(poolName string, propertyList []string) ([]string, error) {
+func GetPoolListPropertyValues(poolName string, propertyList []string) ([]string, error) {
```

Comment:
nit: since it is under pkg/pool/operation, better to not have pool phrase in function name.